examples/morpheusvm/go.mod

```diff
@@ -3,8 +3,8 @@ module github.com/ava-labs/hypersdk/examples/morpheusvm
 go 1.22.8
 
 require (
-	github.com/ava-labs/avalanchego v1.11.13-0.20241230212828-6dea1b366756
-	github.com/ava-labs/hypersdk v0.0.1
+	github.com/ava-labs/avalanchego v1.12.3-warp-verify5.0.20250129181352-a1aff06d0fef
```

Comment:
Why do we have to update avalanchego to use the new `localsigner`? I think we can replace the usage of the hard-coded validator set w/out a version upgrade

Reply:
It's not about the upgrade to `localsigner`. The goal here was to be able to verify the signature `c.Signature.Verify` without providing it with pseudo interface. ( i.e. provide it with the data it truly needs to consume ). see https://github.com/ava-labs/avalanchego/pull/3679 for more details.